jetty-fcgi/fcgi-server/src/main/java/org/eclipse/jetty/fcgi/server/HttpChannelOverFCGI.java

```diff
@@ -43,6 +46,8 @@
 {
     private static final Logger LOG = LoggerFactory.getLogger(HttpChannelOverFCGI.class);
 
+    private final Queue<HttpInput.Content> _contentQueue = new LinkedList<>();
```

Comment:
ALERT ALERT ALERT!!!!!!
OMG a queue!!!!!!!!!!!

:)

Reply:
In FCGI, yes. This makes the implementation sooo much simpler that I don't think it's worth the effort of getting rid of it.